### PR TITLE
cd: use rasactl to operate models

### PR DESCRIPTION
**Pull request by HotThoughts**
Fix https://github.com/RasaHQ/rasa-demo/issues/657

### CODE REVIEW

**Comment by HotThoughts** [comment]
@tczekajlo tag you here because you know rasactl the best. Have you seen [this error](https://github.com/RasaHQ/rasa-demo/runs/4155251456?check_suite_focus=true) before? 
```console
Error: Post "https://***/api/projects/default/models": http2: server sent GOAWAY and closed the connection; LastStreamID=5, ErrCode=ENHANCE_YOUR_CALM, debug=""
```

Is it about the header field size for http2 configured on nginx server?

**Comment by HotThoughts** [comment]
@tczekajlo @melindaloubser1 

💫 Ready for review. See [this](https://github.com/RasaHQ/rasa-demo/runs/4225880413?check_suite_focus=true) successful run. I am so happy that `rasactl` worked here. Love it.

File: .github/workflows/continuous_deployment.yml [hunk 1]
```diff
@@ -146,6 +146,7 @@ jobs:
 
     - name: Prepare namespace
       run: |
+        kubectl label ns "${NAMESPACE}" deployment=sara || true
```
Comment (melindaloubser1) [inline .github/workflows/continuous_deployment.yml:149]: What is this for?

**Review by melindaloubser1 — APPROVED** [review]
Just one Q about what the ns line is for, otherwise glad it works !

File: .github/workflows/continuous_deployment.yml (hunk 1 above)
Comment (HotThoughts) [reply .github/workflows/continuous_deployment.yml:149]: This label is added to ensure that the namespace has at least one label, which helps `rasactl` to request namespace information properly. 

`kubectl` won't give you the requested json data if the namespace does not have any label. 🤷🏻‍♀️

Comment (tczekajlo) [reply .github/workflows/continuous_deployment.yml:149]: It was a workaround, a bug was fixed in the latest version of rasactl. @HotThoughts we can remove this line, it's not needed anymore.

Comment (tczekajlo) [reply .github/workflows/continuous_deployment.yml:149]:
```suggestion
```

Comment (HotThoughts) [reply .github/workflows/continuous_deployment.yml]: Wow. That was fast. Thank you @tczekajlo for fixing it. I will remote this line.

**Pull request merged**